openpype/tools/traypublisher/window.py

```diff
@@ -59,10 +61,17 @@ def __init__(self, publisher_window):
         btns_layout.addWidget(cancel_btn, 0)
         btns_layout.addWidget(confirm_btn, 0)
 
+        txt_filter = QtWidgets.QLineEdit()
```

Comment:
Use `PlaceholderLineEdit` from `openpype.tools.utils` to draw text color properly and please pass `content_widget` as parent.